content/applications/hr/referrals.rst

```diff
@@ -562,13 +407,13 @@ Create rewards
 The rewards are the only configurations needed when setting up the Referrals application. Only users
 with :guilabel:`Administrator` rights for the Recruitment application can create or modify rewards.
 To add rewards, navigate to :menuselection:`Referrals app --> Configuration --> Rewards`. Click
-:guilabel:`Create` and a reward form loads. Enter the following information on the form:
+:guilabel:`New` and a reward form loads. Enter the following information on the form:
 
 - :guilabel:`Product Name`: Enter the name as it should appear for the reward.
```

Comment:
please make sure that all words that immediately follow a ":" are lower-case.
```suggestion
- :guilabel:`Product Name`: enter the name as it should appear for the reward.
```
just fixing this one -- ensure similar formatting follows suit throughout the PR. thanks!